### PR TITLE
[GW] Corrected the sign of the groundwater flow velocity

### PR DESCRIPTION
**Pull request by wenqing**
as titled.

### CODE REVIEW

File: ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h [hunk 1]
```diff
@@ -105,7 +105,7 @@ class LocalAssemblerData
                 ).eval();
 
             for (unsigned d=0; d<GlobalDim; ++d) {
-                _darcy_velocities[d][ip] = darcy_velocity[d];
+                _darcy_velocities[d][ip] = -darcy_velocity[d];
```
Comment (norihiro-w) [inline ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h]: why not add the sign at line 103?

**Comment by chleh** [comment]
Btw. There is a second error in the velocity computation: Since the velocity is computed during assembly, i.e. before the equation system is solved, it will always lag one iteration behind in the output (i.e., one timestep since there is only one iteration per timestep for GWFlow.).

File: ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h (hunk 1 above)
Comment (wenqing) [inline ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h]: Done.

**Comment by wenqing** [comment]
@chleh  Yes, you are right. The velocity calculation must be performed after the linear equation being solved. The error will be corrected after liquid pcs is finished.

**Comment by wenqing** [comment]
@chleh correction. After the non-linear solver.

**Comment by norihiro-w** [comment]
:+1:

**Pull request merged**